types/tx.go

```diff
@@ -26,7 +26,7 @@ type (
 )
 
 // Hash computes the TMHASH hash of the wire encoded transaction.
-func (tx Tx) Hash() []byte {
```

Comment:
This is a breaking change, isn't it?

Reply:
Actually places that use it didn't seem to break for me (see app_test.go change) so it's not a breaking change in this sense. I think I didn't need to make any changes elsewhere since the type HexBytes is a `[]byte` type with methods on it defined.

Reply:
But you have added a `!` to the title, meaning that it is breaking.

I am ok with this change, provided that nothing breaks for outside users.

Reply:
Leave as-is.